pkg/operator/k8sutil/node.go

```diff
@@ -71,15 +72,21 @@ func GetValidNodes(ctx context.Context, rookStorage cephv1.StorageScopeSpec, cli
 	}
 
 	validK8sNodes := []v1.Node{}
+	reasonsForSkippingNodes := map[string][]string{}
```

Comment:
```suggestion
	reasonsForSkippingNodes := map[string]string
```

Why do we need a map of slice? Don't we just want `map["node"]{"reason"}`?

Reply:
It's a map["reason"]"slice of nodes" so that all the nodes with the same reason will print on the same line. If we have a cluster with many nodes I wanted to avoid many lines in the log.

Reply:
Makes sense.